manual/Creating_an_index/Local_indexes/Plain_and_real-time_index_settings.md

```diff
@@ -325,6 +325,17 @@ In plain mode rt_mem_limit can be changed using the following steps:
 * edit rt_mem_limit value in configuration
 * run `ALTER TABLE <index_name> RECONFIGURE`
 
+What you need to know:
+
+*- RT index is very similar to a distributed index of multiple local indexes. The local indexes are called "disk chunks"
+*- rt_mem_limit limits size of the RAM chunk
+*- RAM chunk inside also consists of multiple "plain" indexes, they are called segments
```

Comment:
No. Plain indexes are stored on disk. Segments of ram chunk have nothing related to 'plain' indexes.

Reply:
Edited and improved the description, hopefully it matches the way everything works.